File: BEE 1066.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int num;
        int total_positive = 0;
        int total_negative = 0;
        int total_even = 0;
        int total_odd = 0;
        for(int i = 0; i < 5; i++){
            num = input.nextInt();
            if (num  > 0){
                total_positive += 1;
            }
            if (num < 0){
                total_negative += 1;
            }
            if (num % 2 == 0){
                total_even += 1;
            }
            if (num % 2 == 1){
                total_odd += 1;
            }
        }
        System.out.printf("%d valor(es) par(es)\n", total_even);
        System.out.printf("%d valor(es) impar(es)\n", total_odd);
        System.out.printf("%d valor(es) positivo(s)\n", total_positive);
        System.out.printf("%d valor(es) negativo(s)\n", total_negative);
    }
}

File: BEE 1066.py
total_positive = 0
total_negative = 0
total_even = 0
total_odd = 0
for i in range(5):
    num = int(input())
    if (num > 0):
        total_positive += 1
    if (num < 0):
        total_negative += 1
    if (num % 2 == 0):
        total_even += 1
    if (num % 2 == 1):
        total_odd += 1
print(f"{total_even} valor(es) par(es)")
print(f"{total_odd} valor(es) impar(es)")
print(f"{total_positive} valor(es) positivo(s)")
print(f"{total_negative} valor(es) negativo(s)")